css/base/base.css

```diff
@@ -1,44 +1,56 @@
-html{
+html {
     font-size: 62.5%;
-    
-}
-:root{
---primary-color: #008e76;
---text-color:#0c080c;
---accent-color: #af7a6d;
---bg-color:white;
---alt-text-color: #3E4152;
---complement-btn-text-color: white;
---secondary-color: #0c080c;
-}
-body{
-    padding:0;
-    margin:0;
+
+}
+
+:root {
+    --primary-color: #008e76;
+    --text-color: #0c080c;
+    --accent-color: #af7a6d;
+    --bg-color: white;
+    --alt-text-color: #3E4152;
+    --complement-btn-text-color: white;
+    --secondary-color: #0c080c;
+}
+
+body {
+    padding: 0;
+    margin: 0;
     font-family: 'Roboto', sans-serif;
-    color:var(--alt-text-color);
+    color: var(--alt-text-color);
+}
+
+nav {
+    margin: 0;
+    padding: 0;
 }
-nav{
-    margin:0;
-    padding:0;
+
+* {
+    padding: 0;
+    margin: 0;
 }
```

Comment:
Repeated properties , i think `*` should work. In case of repeated properties you can add multiple selectors.
Eg:
      selector1 ,
      selector2 {
          property : value;
      }
      
Instead of repeating this would be consistent.